support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -122,13 +123,26 @@ const termsAndConditions = css`
 	}
 `;
 
+function checkListFromBenefits(
+	benefits: Array<{ copy: string; tooltip?: string }> | undefined,
+): CheckListData[] | undefined {
+	if (benefits) {
+		return benefits.map((benefit) => ({
+			isChecked: true,
+			text: benefit.copy,
```

Comment:
Potential next PR: We could probably start to align around `text | copy` to avoid these tranformations.

Reply:
See refactor for the moment.